### ISSUE TITLE
Editing Project, Upload Image, not possible to save changes

### ISSUE THREAD

**Issue by mikelovic** [dyne/interfacer-gui#770]
![image](https://github.com/interfacerproject/interfacer-gui/assets/27242634/3317a3b3-7fd9-426a-aa3f-6389c7a32c59)

**Comment by puria** [comment]
@mikelovic Yes known issue 😔 thanks for reporting will try to solve it as soon as we can, and notify it on this issue!
